src/category_theory/limits/fubini.lean

```diff
@@ -238,13 +240,58 @@ begin
   simp, dsimp, simp, -- See note [dsimp, simp].
 end
 
-@[simp]
+@[simp,reassoc]
 lemma limit_iso_limit_curry_comp_lim_inv_π {j} {k} :
   (limit_iso_limit_curry_comp_lim G).inv ≫ limit.π _ (j, k) = limit.π _ j ≫ limit.π _ k :=
 begin
   rw [←cancel_epi (limit_iso_limit_curry_comp_lim G).hom],
   simp,
 end
+end
+
+
+section
+variables [has_limits C] -- Certainly one could weaken the hypotheses here.
+
+open category_theory.prod
+
+/--
+A variant of the Fubini theorem for a functor `G : J × K ⥤ C`,
+showing that $\lim_k \lim_j G(j,k) ≅ \lim_j \lim_k G(j,k)$.
```

Comment:
Is mathjax okay with the `≅` symbol or should this be a tex command?

Reply:
It looks OK to me in the [doc preview](https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%224034%22#docs).

Reply:
Fixed.

Reply:
I don't see any change to the module doc (but it's not a big deal, if you prefer we can fix it later).